packages/next/build/webpack-config.ts

```diff
@@ -1024,6 +1025,23 @@ export default async function getBaseWebpackConfig(
       webpack,
     })
 
+    if (
+      dev &&
+      webpackConfig.devtool &&
```

Comment:
In #13963 it way mentioned that (with respect to changing the sourcemaps):

> unless they're fully disabling them

so here I check to see if `devtool` has a value (which means that the user has set it to something other than `false` (because `false` is okay per that quote above)

Reply:
Sorry, I didn't mean to imply that `false` should be allowed, only that it's the only option with better performance than the current setting!